bin/describe.ml

```diff
@@ -817,6 +1001,7 @@ module What = struct
     | External_lib_deps
     | Opam_files
     | Pp of string
+    | Map
```

Comment:
"map" is a rather general name for a command that does something quite specific. How about "build-path-prefix-map"?

Reply:
Done (in next push)